docs/pages/user_manual.md

```diff
@@ -1,86 +1,154 @@
 # ModbusScope - User Manual
 
-This user manual describes how to get started with *ModbusScope*.
+This user manual describes how to get started with *ModbusScope*. *ModbusScope* is an application which can sample Modbus registers and plot the value in a graph to visualize and investigate the dynamic behavior of a system.
 
-## Installing
+## Overview
+
+The main screen of ModbusScope can contain multiple docked windows. Most are visible at the initial start of the application, except the marker window. This docked windows only appears when a marker has been enabled.
 
-*ModbusScope* can be installed with the provided *.msi* installer. Double-click it and follow the instructions. 
+![image](../_static/user_manual/overview.png)
 
-## Using
+The most important part of the window is the plot view. The data of the active registers that are polled is shown here as plots. The settings in the axis scale windows can be used to visualize a very specific part of the data or auto scale to show all information. The legend shows an overview of the active register. You can hide a specific register plot by double clicking it in the legend. The same action can be used to show the plot again. When a plot is hidden it is still polled, it is only temporary hidden in the plot view. Markers can be used examine the data in a specific time frame. 
+
+## Installing
 
-*ModbusScope* is a program which can sample Modbus registers and plot the value in a graph to visualize and investigate the dynamic behavior of a system.
+*ModbusScope* can be installed with the provided *.msi* installer. Double-click it and follow the instructions. The installer will install the required files on your computer. At the end of the install process, you can also select to open *.mbs* files with *ModbusScope* by default.
```

Comment:
Given Modbusscope is corss-platform. What are the options for installing on Linux/MacOS?

Reply:
None, at the moment. I removed all mentions of it being cross platform from the documentation. Maybe I (or someone else) will add an maintainable installer for Linux or MacOS.